addon/styles/addon.scss

```diff
@@ -1,24 +1,27 @@
+// Internal Styles
```

Comment:
Hm, I didn't realize we don't use ember-component-css. Maybe it didn't work with addons back when ember-osf was new, but [it appears to support addons now](https://github.com/ebryn/ember-component-css#use-in-addons). Not sure if this is beyond the scope of this ticket, but it would be good to use ember-component-css instead of this manual approach.